src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CalculatorTest {
+
+    private static final double INFINITY = 1.0 / 0.0;
+    private Calculator calculator;
+
+    @Before
+    public void setCalculator() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void sum() {
```

Comment:
```suggestion
    public void sumPositiveTest() {
```
the same with others